Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpBuildInfo.cs

```diff
@@ -9,7 +9,6 @@ public class UwpBuildInfo : BuildInfo
     {
         public UwpBuildInfo(bool isCommandLine = false) : base(isCommandLine)
         {
-            BuildPlatform = "x86";
```

Comment:
If no default platform is passed in the parameters, what does it default to?

Reply:
There is no assumed default, the build scripts will just fail to produce any artifacts.

Reply:
Yup, that's a problem.

Reply:
I don't see that it is necessarily. We don't have a default platform for any other build target. Why do we need one for uwp builds?

Reply:
I'm guessing that maybe the problem could be that other folks are depending on the current behavior of the script in their own build setup, so changes to default values can cause inadvertent downstream issues because of these implicit dependencies. If this isn't the case it would be good to call out exactly what the problem is.

If a caller of this script expects that specifying a WSA target would actually build the appx, but then running this script doesn't produce anything (i.e. it succeeds but just doesn't produce an output), then that would be a bad thing. 

However, if there were some log saying that "no BuildPlatform provided, not building an appx output" then it would be clear from the logs that the script was never passed the right info to build.